apps/bikeshare/src/app/components/date-picker/date.component.scss

```diff
@@ -0,0 +1 @@
+@import "~ng-pick-datetime/assets/style/picker.min.css";
```

Comment:
Remove this import declaration. @import basically pastes the entirely of the target CSS file in here which means many declarations will result in a bloated output. On top of that, due to Angular's view encapsulation these styles will not apply to the date picker component nested inside of this component.

Import these styles once in the global styles.scss file.

Reply:
Resolved via c7c8092.